File: CS/WindowsApplication3/RibbonForm.cs
using System;
using System.Collections.Generic;
using System.ComponentModel;
using System.Data;
using System.Drawing;
using System.Text;
using System.Windows.Forms;
using DevExpress.XtraEditors;
using DevExpress.Utils;
using DevExpress.XtraBars.Ribbon;
using DevExpress.XtraBars.Ribbon.ViewInfo;
using DevExpress.Utils.Drawing.Helpers;
using DevExpress.Utils.Drawing;
using DevExpress.XtraBars.Ribbon.Drawing;


namespace DXSample {
    public partial class MyRibbonForm: RibbonForm {
        public MyRibbonForm() {
            InitializeComponent();
           
        }

        internal new bool IsGlassForm
        {
            get { return base.IsGlassForm; }
        }

      

        private void OnButtonPaint(object sender, PaintEventArgs e)
        {
            SimpleButton btn = sender as SimpleButton;
            using (StringFormat format = new StringFormat())
            {
                format.Alignment = StringAlignment.Center;
                e.Graphics.DrawString("->", btn.Font, Brushes.Black, btn.ClientRectangle, format);
            }
        }
    }
}

File: VB/WindowsApplication3/RibbonForm.vb
Imports Microsoft.VisualBasic
Imports System
Imports System.Collections.Generic
Imports System.ComponentModel
Imports System.Data
Imports System.Drawing
Imports System.Text
Imports System.Windows.Forms
Imports DevExpress.XtraEditors
Imports DevExpress.Utils
Imports DevExpress.XtraBars.Ribbon
Imports DevExpress.XtraBars.Ribbon.ViewInfo
Imports DevExpress.Utils.Drawing.Helpers
Imports DevExpress.Utils.Drawing
Imports DevExpress.XtraBars.Ribbon.Drawing


Namespace DXSample
	Partial Public Class MyRibbonForm
		Inherits RibbonForm
		Public Sub New()
			InitializeComponent()

		End Sub

		Friend Shadows ReadOnly Property IsGlassForm() As Boolean
			Get
				Return MyBase.IsGlassForm
			End Get
		End Property



		Private Sub OnButtonPaint(ByVal sender As Object, ByVal e As PaintEventArgs) Handles simpleButton1.Paint, simpleButton2.Paint
			Dim btn As SimpleButton = TryCast(sender, SimpleButton)
			Using format As New StringFormat()
				format.Alignment = StringAlignment.Center
				e.Graphics.DrawString("->", btn.Font, Brushes.Black, btn.ClientRectangle, format)
			End Using
		End Sub
	End Class
End Namespace
